index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul hasan portfolio</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <style> body{background: url("image/pic.png");} </style>
+
+  </head>
+
+  <body>
+    <header>
+    <nav>
+    <div class="navbar">
+            <h1 class="logo"> My Logo </h1>
+
+            <ul>
+              <li class="nav-items"><a href="#">Portfolio</a></li>
+              <li class="nav-items"><a href="#">About</a></li>
+              <li class="nav-items"><a href="#">Contact</a></li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+          </div>
+          </nav>
+        </header>
+
+          <section class="homepage">
+            <div class="intro">
+
+            <h1>I'm Tasnimul <br>Glad to meet you!</h1>
+            <br>
+            <br>
+            <p>
+              I am a software developer. I have done some interesting projects using
```

Comment:
- [ ] This title `I am a software developer.` is a sub-title. In that case, I suggest you place it in it a separate tag like `h2` and above the `p` tag.

```
<h1> // main title </h1>
<h2> // sub-title </h2>
<p> // paragraph </p>
```